src/Umbraco.Infrastructure/Migrations/Upgrade/UmbracoPlan.cs

```diff
@@ -277,6 +278,8 @@ protected void DefinePlan()
             To<UpdateExternalLoginToUseKeyInsteadOfId>("{CA7A1D9D-C9D4-4914-BC0A-459E7B9C3C8C}");
             To<AddTwoFactorLoginTable>("{0828F206-DCF7-4F73-ABBB-6792275532EB}");
 
+            // TO 9.4.0
```

Comment:
```suggestion
            // TO 10.0.0
```

Reply:
This migration was added in 9.4.0 though, the commit introducing it into v10 is a cherrypick from v9